VCFv4.4.draft.tex

```diff
@@ -1701,6 +1706,40 @@ \subsection{Representing unspecified alleles and REF-only blocks (gVCF)}
 \end{flushleft}
 \normalsize
 
+
+\pagebreak
+\subsection{Representing copy number variation}
+\label{cnv}
+
+To encode copy number variation, VCF uses $<$CNV$>$, $<$DEL$>$ and $<$DUP$>$ symbolic structural variant alleles, CN INFO and FORMAT fields.
+
+Allele specific copy number is specified through a $<$CNV$>$ ALT allele for each distinct allelic copy number.
+POS and INFO SVLEN specify the genomic interval over which the copy number is defined.
+As with all symbolic structural variants, the starting position of the interval is the base immediately after POS.
+For example, a region on chr1 from position 101 to 300 (both inclusive) with allele-specific copy numbers of 1 and 2 can be represented as follows:
+
+\footnotesize
+\begin{verbatim}
+	chr1 100 . T <CNV>,<CNV> . . END=130;SVLEN=30,30;CN=1,2 GT:CN 1/2:3
+\end{verbatim}
+\normalsize
+
+All $<$CNV$>$ alleles in the same VCF record should have the same SVLEN.
+To eliminate ambiguity in variant interpretation when GT=0, all copy number haplotypes should be explicitly specified with $<$CNV$>$ alleles, even when copy number is 1.
+
+If only total copy number is known, the copy number of the segment should be defined with a single $<$CNV$>$ ALT allele with a missing INFO CN field.
+In the above example this corresponds to the following:
+
+\footnotesize
+\begin{verbatim}
+	chr1 100 . T <CNV> . . END=130;SVLEN=30 GT:CN ./.:3
```

Comment:
Could `<DUP>` be used here instead of `<CNV>` since we know there must be a duplication (i.e. CN>1) haplotype given that the total CN for the diploid region is 3.

Could we then refer to the `<DUP>` haplotype in the GT field as `./1`? This would give a more clear indication that the sample is variant and not just reporting an unknown genotype?

Reply:
`<DUP>` `./1` only works if you make a haploid or diploid assumption. When dealing with genomic rearrangements, these assumptions are implicitly violated. In general, my preference is for the VCF to match the claim being made as closely as possible. A CN=3 claim is subtly different than a `<DUP> `./1` claim (e.g. the latter could be based on a single molecule containing multiple copies of a given region - (e.g. from ONT, optical mapping or flow-sorting individual chromosomes)).